packages/quantic/force-app/main/default/lwc/quanticPager/e2e/fixture.ts

```diff
@@ -0,0 +1,28 @@
+import {PagerObject} from './page-object';
+import {quanticBase} from '../../../../../../playwright/fixtures/base-fixture';
+
+import {SearchObject} from '../../../../../../playwright/page-object/search-object';
+
+type QuanticPagerE2EFixtures = {
+  pager: PagerObject;
+};
+
+export const testSearch = quanticBase.extend<QuanticPagerE2EFixtures>({
+  pager: async ({page}, use) => {
+    await use(new PagerObject(page));
+  },
+  search: async ({page}, use) => {
+    await use(new SearchObject(page, 'search'));
+  },
+});
+
+export const testInsight = quanticBase.extend<QuanticPagerE2EFixtures>({
+  pager: async ({page}, use) => {
+    await use(new PagerObject(page));
+  },
+  search: async ({page}, use) => {
```

Comment:
should this be `insight` (granted that you actually need two fixtures)?

Reply:
this has been addressed, https://github.com/coveo/ui-kit/pull/4597#discussion_r1823268866